### ISSUE TITLE
Importing the .json into Tableau. Need help!

### ISSUE THREAD

**Issue by elina-nesterova** [simzou/nielsen-dma#10]
Can anyone tell me if its possible to import the .json into Tableau for visualization (doing a social marketing campaign for a TV show client). ? I tried, but didnt get the result I expected. Thank you!

**Comment by stevengritz** [comment]
Yes, it is possible. Just change the extension to a .geojson file so Tableau will recognize it.
